02-microarray/pathway-analysis_microarray_03_gsea.Rmd

```diff
@@ -437,27 +452,34 @@ Let's look for the gene set with the highest negative NES.
 
 ```{r}
 gsea_result_df %>%
-  # The `dplyr::arrange()` function sorts the values in ascending order
-  dplyr::arrange(NES)
+  # Use `top_n()` to return the top n observation using `NES` as the ordering
+  # variable -- note the use of `dplyr::desc()` to sort the values in descending order
+  dplyr::top_n(n = 1, wt = dplyr::desc(NES))
 ```
 
-The gene set `HALLMARK_E2F_TARGETS` has a highly negative NES.
+The gene set `HALLMARK_E2F_TARGETS` has the highest negative NES.
 
 ```{r}
-enrichplot::gseaplot(gsea_results,
+highly_negative_nes_plot <- enrichplot::gseaplot(
+  gsea_results,
   geneSetID = "HALLMARK_E2F_TARGETS",
   title = "HALLMARK_E2F_TARGETS",
   color.line = "#0d76ff"
 )
+
+highly_negative_nes_plot
 ```
 
 This gene set shows the opposite pattern -- genes in the pathway tend to be on the right side of the graph.
+Again, the red dashed line here indicates the peak ES score (the ES is the maximum deviation from zero).
```

Comment:
This would probably be
```suggestion
Again, the red dashed line here indicates the maximum deviation from zero, in other words, the most negative ES score.
```